src/test/dynamo/Beyonce.test.ts

```diff
@@ -426,7 +432,38 @@ async function testChunkedBatchGet(jayZ?: JayZ) {
     keys: songs.map(({ id, musicianId }) => SongModel.key({ id, musicianId }))
   })
 
-  expect(sortById(results.song)).toEqual(songs)
+  expect(sortById(results.items.song)).toEqual(songs)
+  expect(results.unprocessedKeys).toEqual([])
+}
+
+async function testBatchGetWithUnprocessedKeys() {
```

Comment:
For context Dynamo typically responds with `unprocessedKeys` if you ask for too much data  OR you have a table in provisioned capacity mode and exceed your capacity. 

The _local_ version of DynamoDB _ignores_ provisioned capacity settings, so this is the easiest way to trigger this behavior.

Reply:
> The local version of DynamoDB ignores provisioned capacity settings, so this is the easiest way to trigger this behavior

Hmm, since [we're using the local version of DynamoDB (or in Docker if we're on CI)](https://github.com/ginger-io/beyonce/blob/master/src/test/dynamo/util.ts#L30), I don't follow what's causing some keys to be unprocessed here, given that local DynamoDB doesn't respect provisioned capacity settings.

Reply:
Ah this test works because we ask for more than 16MB of data, which is a distinct behavior from enforcing provisioned capacity settings (e.g. how many reads/writes a second you attempt to do).

Reply:
Perfect, that makes sense!